lib/dynamo/dynamo/jobs.py

```diff
@@ -118,6 +143,34 @@ def _prepare_job_for_database(
     return prepared_job
 
 
+def _prepare_archive_job_for_database(job: dict, user_id: str, request_time: str, product: asf.ASFProduct) -> dict:
+    prepared_job = {
+        'job_id': str(uuid4()),
+        'user_id': user_id,
+        'status_code': 'SUCCEEDED',
+        'execution_started': True,
+        'request_time': request_time,
+        'processing_times': [0],
+        'credit_cost': 0,
```

Comment:
It shouldn't matter here, but since credits represent hyp3's "currency", it's good practice to always represent them as `Decimal` objects so as to avoid any floating point precision issues.
```suggestion
        'credit_cost': Decimal(0),
```